src/main/java/com/android/volley/toolbox/AsyncHttpStack.java

```diff
@@ -57,15 +59,27 @@ public abstract void executeRequest(
      * This method sets the non blocking executor to be used by the stack for non-blocking tasks.
      * This method must be called before executing any requests.
      */
-    @RestrictTo({RestrictTo.Scope.SUBCLASSES, RestrictTo.Scope.LIBRARY_GROUP})
-    public abstract void setNonBlockingExecutor(ExecutorService executor);
+    @RestrictTo({RestrictTo.Scope.LIBRARY_GROUP})
+    public void setNonBlockingExecutor(ExecutorService executor) {
+        mNonBlockingExecutor = executor;
+    }
 
     /**
      * This method sets the blocking executor to be used by the stack for potentially blocking
      * tasks. This method must be called before executing any requests.
      */
-    @RestrictTo({RestrictTo.Scope.SUBCLASSES, RestrictTo.Scope.LIBRARY_GROUP})
-    public abstract void setBlockingExecutor(ExecutorService executor);
+    @RestrictTo({RestrictTo.Scope.LIBRARY_GROUP})
+    public void setBlockingExecutor(ExecutorService executor) {
+        mBlockingExecutor = executor;
+    }
+
+    protected ExecutorService getBlockingExecutor() {
```

Comment:
javadoc for this and getNonBlockingExecutor

Reply:
I think this javadoc wasn't included in your last commit, please add, thanks!